tests/migrations/test_deprecated_fields.py

```diff
@@ -0,0 +1,22 @@
+from django.core.management import call_command
+from django.test import override_settings
+
+from .test_base import MigrationTestBase
+
+
+class Tests(MigrationTestBase):
+    """
+    Deprecated model fields should still be usable in historic migrations.
+    """
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.deprecated_field_migrations"})
+    def test_migrate(self):
+        # Make sure no tables are created
+        self.assertTableNotExists("migrations_ipaddressfield")
+        # Run migration
+        call_command("migrate", verbosity=0)
```

Comment:
Now that `IPAddressField` doesn't have an `get_internal_type()` anymore, the created field is a varchar?